.eslintrc

```diff
@@ -9,6 +9,8 @@ ecmaFeatures:
   binaryLiterals: true
   generators: true
   forOf: true
+  objectLiteralShorthandMethods: true
+  objectLiteralShorthandProperties: true
```

Comment:
used here: https://github.com/nodejs/io.js/pull/1771/files#diff-7718507eb7a71106a68401cae76c5268R14

I don't see a good reason not to enable these other than backporting, maybe.